mmv1/templates/terraform/examples/os_login.tf.erb

```diff
@@ -0,0 +1,50 @@
+# [START compute_terraform-oslogin-example]
+
+# [START compute_enable_oslogin_api]
+resource "google_project_service" "project" {
+  service = "oslogin.googleapis.com"
```

Comment:
Test currently fails because after the test runs it attempts to delete all resources created. This resource is attempted to be deleted which disables the service. There are many dependencies on the service so it gets denied.

I think other tests get around this by setting
```
  disable_on_destroy         = false
```
on the project service resource